docs/config/README.md

```diff
@@ -171,6 +171,27 @@ MINIO_ETCD_CLIENT_CERT_KEY  (path)      client cert key for mTLS authentication
 MINIO_ETCD_COMMENT          (sentence)  optionally add a comment to this setting
 ```
 
+### API
+
+By default, there is no limitation on the number of concurrents requests that a server/cluster processes at the same time. However, it is possible to impose such limitation using the API subsystem. Read more about throttling limitation in MinIO server [here](https://github.com/minio/minio/blob/master/docs/throttle/README.md).
+
+```
+KEY:
+api  tune requests throttling
+
+ARGS:
+requests-max       (number)     set the maximum number of concurrent requests
```

Comment:
This should be `requests_max` we do not follow `-` anymore and the other one is `request_deadline`